test_runner/regress/test_tenant_conf.py

```diff
@@ -1,19 +1,29 @@
 from contextlib import closing
 
 import psycopg2.extras
+import pytest
 from fixtures.log_helper import log
 from fixtures.neon_fixtures import NeonEnvBuilder
 
 
 def test_tenant_config(neon_env_builder: NeonEnvBuilder):
+    """Test per tenant configuration"""
     # set some non-default global config
     neon_env_builder.pageserver_config_override = """
 page_cache_size=444;
 wait_lsn_timeout='111 s';
 tenant_config={checkpoint_distance = 10000, compaction_target_size = 1048576}"""
 
     env = neon_env_builder.init_start()
-    """Test per tenant configuration"""
+
+    # Check that we raise on misspelled configs
+    with pytest.raises(Exception):
```

Comment:
NIT: maybe worth checking, that we're failing due to the intended reason and `some_invalid_setting_name_blah_blah` is contained in the exception message